contributing/contributing-code.md

```diff
@@ -149,6 +149,8 @@ It is acceptable for one person to fulfill these requirements, e.g. if an Owner
 
 We now have a bot that will automatically assign Owners to a PR once it is created, and it is likely at least one of these Owners will also be a Reviewer.
 
+Depending on the files/folders involved, Owners include one or more [AMP Working Groups](https://amp.dev/community/working-groups/access-subscriptions/), such as the groups for  [Security & Privacy](https://amp.dev/community/working-groups/security-privacy/) and [User Interface & Accessibility](https://amp.dev/community/working-groups/ui-and-a11y/).
```

Comment:
This is a better link for `AMP working groups`: https://amp.dev/community/governance/#working-groups

Reply:
Ah, perfect. Yes that's a much better link (I was going by the link in the navigation that you get to for "Community > Working Groups" - it might be worth, separately, setting up a high-level page that's generic for all WGs there, rather than linking straight to the first WG)